src/library_test_contracts/SafeMathUint256Tester.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+contract SafeMathUint256Tester {
+    function initialize() public constant {
+    }
+
+    function mul(uint256 _a, uint256 _b) public constant returns (uint256) {
+        return SafeMathUint256.mul(_a, _b);
```

Comment:
The syntax for these can be `_a.mul(_b)` by putting `using SafeMathUint256 for uint256` at the top of the contract.  Recommend using this style as this is how we will be using the library, in case there are any oddities with that usage pattern.